components/common/resources/ome/config.xml

```diff
@@ -169,6 +169,16 @@
        <property name="mutable" value="false"/>
        <property name="visibility" value="all"/>
     </bean>
+    <bean class="ome.system.Preference" id="omero.client.ui.tree.orphans">
+       <property name="db" value="false"/>
+       <property name="mutable" value="false"/>
+       <property name="visibility" value="all"/>
+    </bean>
+    <bean class="ome.system.Preference" id="omero.client.ui.tree.orphans.enabled">
```

Comment:
``omero.client.ui.tree.orphans`` is wrapper to reduce number of calls and shouldn't being touch. People should use only ``.enabled``, ``.name``, and ``.description``. Documentation will be updated accordingly.